tests/test_networks.py

```diff
@@ -13,12 +13,17 @@ async def test_list_networks(docker):
 
 @pytest.mark.asyncio
 async def test_list_networks_with_filter(docker):
-    await docker.networks.create({
+    network = await docker.networks.create({
         "Name": "test-net-filter",
-        "Labels": {"some": "label"},
+        "labels": {"some": "label"},
```

Comment:
According to the docker API reference, this field starts with an uppercase letter, though Docker seems to accept both lowercase and uppercase letters: https://docs.docker.com/engine/api/v1.45/#tag/Network/operation/NetworkCreate

Reply:
This was an accident, I'll revert.